dotcom-rendering/src/components/MostPopularFooterGrid.tsx

```diff
@@ -105,6 +105,7 @@ export const MostPopularFooterGrid = ({
 	return (
 		<div
 			data-component={ophanLinkName(sectionName)}
+			data-link-name={ophanLinkName(sectionName)}
```

Comment:
Adds this

![image](https://github.com/guardian/dotcom-rendering/assets/19683595/726016c4-e6c0-43fd-ac3b-a581bdbe4edf)

but will revisit the whole Most Popular section. The DOM has diverged significantly between frontend and DCR after Deeply Read was added:

| frontend | DCR |
|--------|-------|
| <img width="1082" alt="image" src="https://github.com/guardian/dotcom-rendering/assets/19683595/b29040ea-48b3-425e-8089-96ddf1b2ef6a"> | ![image](https://github.com/guardian/dotcom-rendering/assets/19683595/1dee6516-457a-4304-886a-b3a752f0b7be) |